samples-generator/bin/3d-tiles-samples-generator.js

```diff
@@ -252,7 +258,8 @@ var promises = [
     createCompositeOfComposite(),
     createCompositeOfInstanced(),
     // Hierarchy
-    createHierarchy(),
+    createHierarchyExtension(),
+    createLegacyHierarchy(),
     createHierarchyMultipleParents(),
     createHierarchyNoParents(),
     createHierarchyBinary(),
```

Comment:
Just nitpicking on names. I think it's fine to drop `Extension` since the other hierarchy samples don't have that. And for symmetry, change `createLegacyHierarchy` to `createHierarchyLegacy`.

```
createHierarchyExtension
createLegacyHierarchy
createHierarchyMultipleParents
createHierarchyNoParents
createHierarchyBinary
```
to
```
createHierarchy
createHierarchyLegacy
createHierarchyMultipleParents
createHierarchyNoParents
createHierarchyBinary
```